vsintegration/src/FSharp.Editor/Completion/CompletionProvider.fs

```diff
@@ -45,7 +45,7 @@ type internal FSharpCompletionProvider
         |> List.filter (fun (keyword, _) -> not (PrettyNaming.IsOperatorName keyword))
         |> List.sortBy (fun (keyword, _) -> keyword)
         |> List.mapi (fun n (keyword, description) ->
-             CommonCompletionItem.Create(keyword, Nullable Glyph.Keyword, sortText = sprintf "%06d" (1000000 + n))
+             CommonCompletionItem.Create(keyword, CompletionItemRuls.Default, Nullable Glyph.Keyword, sortText = sprintf "%06d" (1000000 + n))
```

Comment:
I never said I could spell :smile:

Once we have an official package I can build against I'll do a local build to catch anything outstanding.